### PR TITLE
Add instructions for conversation ID and permissions.

### PR DESCRIPTION
**Pull request by scottdangelo**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -62,12 +62,20 @@ If you followed the previous steps you should already be inside your local clone
...
 7. Select `Advanced Mode` in the configuration window.
-8. Click `Add Variable` and name your new variable `ConversationV1_ID` then set its value to the Workspace ID of your Conversation workspace.
-    ![Variable Configuration Example](doc/source/images/add_variable.png?raw=true)
+8. Open the script vr-speech-sandbox-cardboard/SpeechSandbox/Assests/Scripts/VoiceSpawner.cs and put your workspace ID on line #34 in the Start() method.
  You can find your workspace ID by selecting the expansion menu on your conversation workspace and selecting `View details`.
     ![View Details Location](doc/source/images/workspace_details.png?raw=true)
 9. In the Unity editor project tab, select Assets->Scenes->MainGame->MainMenu and double click to load the scene.
 10. Press Play
+11. To Build an android .apk file and deploy to your phone, you can File -> Build Settings (Cntrl + Shift +B) and click Build. When prompted you can name your build and then move it to your phone.
```
Comment (markstur) [inline README.md]: s/Cntrl/Ctrl/  I think is more common.  Probably `CTRL` even if you do that consistently with other stuff too

File: README.md [hunk 2]
```diff
@@ -62,12 +62,20 @@ If you followed the previous steps you should already be inside your local clone
...
+8. Open the script vr-speech-sandbox-cardboard/SpeechSandbox/Assests/Scripts/VoiceSpawner.cs and put your workspace ID on line #34 in the Start() method.
  You can find your workspace ID by selecting the expansion menu on your conversation workspace and selecting `View details`.
     ![View Details Location](doc/source/images/workspace_details.png?raw=true)
 9. In the Unity editor project tab, select Assets->Scenes->MainGame->MainMenu and double click to load the scene.
 10. Press Play
+11. To Build an android .apk file and deploy to your phone, you can File -> Build Settings (Cntrl + Shift +B) and click Build. When prompted you can name your build and then move it to your phone. 
+Alternately, connect the phone via USB and File-> Build and Run (or Control+B).
+    Make sure you have enabled USB Debugging:
+    A. Open Settings-> About-> Software Information-> More.
+    B. Then tap �Build number� seven times to enable Developer options. ...
```
Comment (markstur) [inline README.md]: not sure why period and then ...

File: README.md (hunk 1 above)
Comment (markstur) [inline README.md]: the needs more md formatting to look good in the readme

Comment (scottdangelo) [inline README.md]: good call

Comment (scottdangelo) [inline README.md]: yes. need an md preview editor...

File: README.md (hunk 2 above)
Comment (scottdangelo) [inline README.md]: because of copy-n-paste.

**Pull request merged**